accounts/service.go

```diff
@@ -672,6 +672,16 @@ func (s *InterceptorService) TrackPayment(id AccountID, hash lntypes.Hash,
 					log.Debugf("Payment %v not initiated, "+
 						"stopping tracking", hash)
 
+					s.Lock()
+					defer s.Unlock()
+
+					// We also remove the payment from the
+					// pending payments map, so that the
+					// payment won't be seen as in-flight
+					// balance when calculating the
+					// account's available balance.
+					delete(s.pendingPayments, hash)
```

Comment:
Why do we keep the payment request in the accounts? Could we alternatively call `s.removePayment`? If a payment wasn't initiated we should track it the next time it is really attempted, not?